flake.nix

```diff
@@ -6,6 +6,12 @@
     flake-utils.url = "github:numtide/flake-utils";
     rust-overlay.url = "github:oxalica/rust-overlay";
     naersk.url = "github:nmattia/naersk";
+    helix = {
+      flake = false;
+      url = "https://github.com/helix-editor/helix";
+      type = "git";
```

Comment:
Consider replacing it with `url = github:helix-editor/helix;`

Reply:
Thanks for the review. I actually tried that, as well as `"git:https://..."`, but for some reason it on doesn't understand the submodules attribute when doing so. This is the only way I got it to work.